### DISCUSSION TITLE
Is there a way to enable frame dropping?

### DISCUSSION THREAD

**Discussion by JamesJunsungKim** [HaishinKit/HaishinKit.swift#1152] [Q&A] [answered]
As far as I know, there used to be a way to disable auto frame dropping in the past but with the recent update, it's default setting to not to drop frames in a poor connection.

my question is to know if there is a way to enable frame dropping?

**Comment by shogo4405** [comment, marked as answer]
Not. You probably want to use it for bandwidth control, but it was better to change the bitrate, so I removed it. Are you in trouble?
